web_src/js/features/projects.js

```diff
@@ -37,26 +36,34 @@ export default async function initProject() {
     },
   );
 
+  const moveIssue = (e) => {
+    const boardCards = e.to.getElementsByClassName('board-card');
+
+    boardCards.forEach((card, i) => {
+      const id = $(card).data('issue');
+      $.ajax(`${e.to.dataset.url}/${id}/${i}`, {
```

Comment:
we need a batch update, but multiple requests.

Reply:
Sorting is asc. Top is `0` and bottom should be `n`, but after first adding multiple issues they will all start with `0`.
I see two options for batch updates:
- `UPDATE project_issue SET sorting = sorting + 1 WHERE sorting > ? ` which would increment any following issues of the moved one. Problem with this is how should the initial state with all issues having `0` be handled? Maybe something like this example: https://stackoverflow.com/a/4975383/8461267
- Change the ajax request to be send once with a json list of `issue-id => sorting` mapping and save that via a batch update.

Reply:
I vote for the second one, it works for legacy data whose `sorting` is 0.

Reply:
@wxiaoguang I updated it to use batch updates (hopefully I used the right way of doing it). I have a problem with my nested array not being captured from `MoveProjectIssuesForm`. How could I fix that? Are nested form parts even a thing?

Reply:
Maybe you need something like `json:"issues",binding:"Required"`?

Reply:
I used `map[string]` instead of a simple array `[]` 🤦🏾 
It seems to work just fine for me now.